sherlocked.js

```diff
@@ -0,0 +1,74 @@
+var sherlocked = require('sherlocked');
+
+var PORT = require('./config').PORT;
+
+var environments = [
+    {
+        browserName: 'firefox',
+        platform: 'OS X 10.9'
+    },
+    {
+        browserName: 'chrome',
+        platform: 'OS X 10.9'
+    }
+];
+
+
+var captures = [
+    {
+        name: 'Home Page on Mobile',
+        capture: function(client) {
+            return client.setViewportSize({width: 320, height: 960})
+                .url('http://localhost:' + PORT)
+                .waitForExist('.feed-home', 180000);
+        }
+    },
+    {
+        name: 'Home Page on Desktop',
+        capture: function(client) {
+            return client.setViewportSize({width: 1050, height: 2048})
```

Comment:
I'd suggest pulling the `setViewportSize` and `url` calls into a helper function.

```js
function startDesktop(client, path) {
    return client.setViewportSize({width: 1050, height: 2048})
                 .url('http://localhost:' + PORT + path);
}
// ...
    capture: function(client) {
        return startDesktop(client, '/').waitForExist('.feed-home', 180000);
    },